### PR TITLE
version added to pom files

### PR DESCRIPTION
**Pull request by karthikbr82**
https://www.pivotaltracker.com/story/show/118628191

### CODE REVIEW

**Comment by karthikbr82** [comment]
@eljasala PTAL

File: callstats-java-sdk/pom.xml [hunk 1]
```diff
@@ -1,10 +1,38 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>callstats</groupId>
+  <groupId>io.callstats</groupId>
   <artifactId>callstats</artifactId>
```
Comment (eljasala) [inline callstats-java-sdk/pom.xml]: Change to callstats-java-sdk

File: callstats-java-sdk/pom.xml [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
   <packaging>jar</packaging>
-  <version>0.0.1-SNAPSHOT</version>
+  <version>3.1.0</version>
```
Comment (eljasala) [inline callstats-java-sdk/pom.xml]: Add -SNAPSHOT, it will be removed when we do releases.

**Comment by eljasala** [comment]
Format the pom.xml file, there are different indentations. Other than that & comments, LGTM

**Pull request merged**